src/Components/error-boundary/error-boundary.component.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+
+import { ErrorImageOverlay, ErrorImageContainer, ErrorImageText } from './error-boundary.styles';
+
+export class ErrorBoundary extends Component {
+	constructor () {
+		super();
+
+		this.state = {
+			hasErrored: false,
+		};
+	}
+
+	static getDerivedStateFromError (error) {
+		return { hasErrored: true };
+	}
+
+	componentDidCatch (error, info) {
+		// console.log(error);
```

Comment:
Whats the purpose of this componentDidcatch, plus its empty

Reply:
Mainly, purpose for catching and consol log the error for the development. I commented console log because we dont want to show the error in production